db/migrate/20221021183414_create_lesson_previews.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLessonPreviews < ActiveRecord::Migration[6.1]
+  def change
+    create_table :lesson_previews, id: :uuid do |t|
+      t.text :content, default: '', null: false
```

Comment:
Can we remove the default here please? we don't ever want to have lesson previews with empty content.